The bumper workflow miss the `--admin` flag

The bumper worflow is missing the `--admin` flag so the `wazuhci` user cannot merge the bump PR 

- https://github.com/wazuh/wazuh-dashboard-ml-commons/actions/runs/18030285198/job/51305090749
- https://github.com/wazuh/wazuh-dashboard-ml-commons/pull/17

```
Run # Any checks for the PR are bypassed since the branch is expected to be functional (i.e. the bump process does not introduce any bugs)
X Pull request wazuh/wazuh-dashboard-ml-commons#17 is not mergeable: the base branch policy prohibits the merge.
To have the pull request merged after all the requirements have been met, add the `--auto` flag.
To use administrator privileges to immediately merge the pull request, add the `--admin` flag.
Error: Process completed with exit code 1.
```

Example:

https://github.com/wazuh/wazuh-dashboard/blob/1bef5943142fca1f9bed4c0255709065ff98b2a6/.github/workflows/4_bumper_repository.yml#L147

--- a/.github/workflows/4_bumper_repository.yml
+++ b/.github/workflows/4_bumper_repository.yml
@@ -124,7 +124,7 @@ jobs:
       - name: Merge pull request
         run: |
           # Any checks for the PR are bypassed since the branch is expected to be functional (i.e. the bump process does not introduce any bugs)
-          gh pr merge "${{ steps.create_pr.outputs.pull_request_url }}" --merge
+          gh pr merge "${{ steps.create_pr.outputs.pull_request_url }}" --merge --admin
 
       - name: Show logs
         run: |

